security/security-spring/src/main/resources/applicationContext-security.xml

```diff
@@ -533,6 +567,7 @@
     <!-- authenticate is off but we have session-service beans -->
     <b:beans profile="noauthsessionservice">
         <global-method-security pre-post-annotations="disabled"/>
+        <b:bean id="restAuthenticationEntryPoint" class="org.cbioportal.security.spring.sessionservice.RestAuthenticationEntryPoint"/>
```

Comment:
For some reason if "noauthsessionservice" was selected, it would throw an error that it couldn't find the "restAuthenticationEntryPoint" bean, even though it is globally defined.  I don't think any portals are using the "noauthsessionservice" option though, I think session service is working with authentication and authorization = false, so perhaps we don't need this profile anymore.  Anyway maybe this PR is not the place to add this, because it is not related to the token issue.

Reply:
No, its defined only when authentication is turned on:

 <b:beans profile="googleplus,openid,saml,ad,ldap">
...
</b:beans>

Reply:
I was about to merge this .. but I will wait for the update to spring-security v5